test/xbgpu/test_engine.py

```diff
@@ -498,7 +494,7 @@ async def xbengine(
         [None, 0, 3],
         filter=valid_end_to_end_combination,
     )
-    @pytest.mark.parametrize("heap_accumulation_threshold", test_parameters.heap_accumulation_threshold)
+    @pytest.mark.parametrize("heap_accumulation_threshold", [[3, 7]])
```

Comment:
I know greater context and priorities, but probably worth changing this to a list of tuples (rather than a list of lists) - even if only for readability.